src/templates/layout/base.twig

```diff
@@ -27,6 +27,20 @@
                         <li><a href="{{ url('watch.list') }}">Watch Functions</a></li>
                         <li><a href="{{ url('waterfall.list') }}">Waterfall</a></li>
                     </ul>
+                    {% set sites = sites() %}
```

Comment:
Another option would be to add the sites into the extension, [when the extensions are added to twig](https://github.com/perftools/xhgui/blob/master/src/Xhgui/ServiceContainer.php#L59).